src/resources/i18n/translations/en.json

```diff
@@ -180,30 +181,30 @@
   "Auto": "Auto",
   "Auto Downloading Off": "Auto-downloading off",
   "Auto Downloading On": "Auto-downloading on",
-  "Auto queue position helper text": "Set where new episodes should be added to your queue when auto-queue is enabled for a podcast.",
-  "Automatically add new episodes to queue": "Automatically add new episodes to queue",
+  "Auto queue position helper text": "Only has an impact if auto-queueing is turned on for a podcast.",
+  "Automatically add new episodes to queue": "Auto-enqueue new episodes",
   "Autoscroll Off": "Auto-scroll off",
   "Autoscroll On": "Auto-scroll on",
   "Back": "Back",
   "Back To App": "Back To App",
   "Back To Login": "Back To Login",
   "Balance": "Balance",
-  "Bitcoin Wallet": "Bitcoin Wallet",
+  "Bitcoin Wallet": "Bitcoin wallet",
   "Boost": "Boost",
   "Boost Amount": "Boost amount",
   "Boost Amount for this Podcast": "Boost amount for this podcast",
   "Boost Sent": "Boost sent",
   "Boost splits": "Boost splits",
   "brandName chat is powered by": "Podverse chat uses these libre libraries",
   "Bug Report": "Report a bug",
-  "Buy a Podverse premium membership": "Buy Podverse Premium for someone",
+  "Buy a Podverse premium membership": "Gift Podverse Premium to someone",
   "by": "by:",
   "By": "By:",
   "cancel": "cancel",
   "Cancel": "Cancel",
   "Cancelled": "Cancelled",
   "Category": "Category",
-  "Censor NSFW text": "Censor NSFW text",
+  "Censor NSFW text": "Remove censorable text",
```

Comment:
NSFW is widely understood in the US to mean "not safe for work" aka "offensive". I'm going to change this back...

However, I can see NSFW causing confusion for translators. I've been thinking of starting a context document, where I list the problematic base language text and add an additional explanation of its context. 

I'm not sure how else to do this but just a txt or md file. Somehow translators would need to be informed that the document exists. If someone asks a question about what the base language text means, we can make a good habit of adding some context to the document for that text. If you have any better ideas for handling please lmk.

Reply:
Translator comments can be added to the right-hand field of each string in Weblate. Context as screenshots is also possible.